src/scripts/seen_posts.js

```diff
@@ -52,11 +52,15 @@ const lengthenTimelines = () =>
 const dimPosts = function (postElements) {
   lengthenTimelines();
 
-  for (const postElement of filterPostElements(postElements, { excludeClass, timeline, includeFiltered })) {
+  for (const postElement of filterPostElements(postElements, { timeline, includeFiltered })) {
     const { id } = postElement.dataset;
+    const timelineItem = getTimelineItemWrapper(postElement);
+
+    if (timelineItem.getAttribute(excludeAttribute) === '') return;
```

Comment:
would prefer explicitly checking for existence of the attribute, rather than a specific value

```suggestion
    if (timelineItem.getAttribute(excludeAttribute) !== null) return;
```